.gitignore

```diff
@@ -7,3 +7,4 @@ pom.xml.asc
 *.class
 /.lein-*
 /.nrepl-port
+out/
```

Comment:
This line can be removed if we use the target directory for ClojureScript output.